templates/server.xml.j2

```diff
@@ -136,7 +136,11 @@
         <Valve className="org.apache.catalina.valves.AccessLogValve" directory="{{ path }}"
                prefix="localhost_access_log" suffix=".txt"
                pattern="%h %l %u %t &quot;%r&quot; %s %b" />
-
+        
+        {% if tomcat_show_server_info_on_error_page == false %}
```

Comment:
A better condition would be`if not tomcat_show_server_info_on_error_page`

Reply:
hmm i like it to be explicit, that would make any value on `tomcat_show_server_info_on_error_page` as valid

Reply:
if wont, that would be `tomcat_show_server_info_on_error_page is defined`